src/components/molecules/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -176,6 +183,11 @@ export default {
 
     #oc-breadcrumb-contextmenu-trigger > span {
       vertical-align: middle;
+      border: 3px solid transparent;
+      &:hover {
+        background-color: var(--oc-color-background-secondary);
```

Comment:
Doesn't seem to be doing anything in the example?

Reply:
ah damn forgot about lightmode, in darkmode its fine but secondary in light mode is also white